### PR TITLE
Fix bug/3 and bug/4

### PR DESCRIPTION
**Pull request by alexeypato**
@AlehZaksheuski

### CODE REVIEW

File: src/inputs/Input.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.add__btn {
```
Comment (OlegZaksheuski) [inline src/inputs/Input.css]: Please, use correct classname with CSS-style. For example:
'add-button' or 'add-btn'

File: src/inputs/Input.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	border: 1px solid #333; 
+    padding: 5px;
```
Comment (OlegZaksheuski) [inline src/inputs/Input.css]: Please, use the same alignment. (If github incorrect view align - all ok)

File: src/inputs/Input.jsx [hunk 3]
```diff
@@ -30,13 +32,13 @@ export default class Input extends React.Component {
           value={this.state.value}
           ref='cash'
         />
-        <button
+        <a
           className='add__btn'
           onClick={() => this.handleAddCash()}
-          ref='alert_button'
+          href='#'
```
Comment (OlegZaksheuski) [inline src/inputs/Input.jsx]: If you click this link, you move to top of page. Correct link:
href = {undefined}

File: src/inputs/Input.jsx [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 import ReactDOM from 'react-dom';
+import './Input.css';
 
 export default class Input extends React.Component {
-
   constructor(props) {
     super(props);
     this.state = {
-      value: '',
+      value: ''
```
Comment (OlegZaksheuski) [inline src/inputs/Input.jsx]: Why you delete comma?

File: src/inputs/Input.jsx [hunk 5]
```diff
@@ -17,8 +17,10 @@ export default class Input extends React.Component {
   };
 
   handleAddCash = () => {
-    this.props.addCash(this.state.value);
-    this.setState({ value: '' });
+    if (this.state.value.trim().length > 0) {
```
Comment (OlegZaksheuski) [inline src/inputs/Input.jsx]: Please, add error message for user.

**Comment by OlegZaksheuski** [comment]
@alexeypato Fix please.

File: src/inputs/Input.css [hunk 6]
```diff
@@ -0,0 +1,12 @@
+.add__btn {
+	border: 1px solid #333; 
+    padding: 5px; 
+    text-decoration: none !important;
```
Comment (OlegZaksheuski) [inline src/inputs/Input.css]: Dont use important!

File: src/inputs/Input.css (hunk 2 above)
Comment (alexeypato) [inline src/inputs/Input.css]: github incorrect view align - all ok

File: src/inputs/Input.css (hunk 6 above)
Comment (alexeypato) [inline src/inputs/Input.css]: Here it is necessary to use !important, differently isn't blocked reference underlining

**Pull request closed without merging**